.travis_mariadb.sh

```diff
@@ -1,16 +0,0 @@
-#!/usr/bin/env bash
-
-set -eu
-
-service mysql stop
-apt-get purge '^mysql*' 'libmysql*'
-apt-key adv --recv-keys --keyserver hkp://keyserver.ubuntu.com:80 0xcbcb082a1bb943db
-
-if [[ x$1 = xmariadb55 ]]; then
-  add-apt-repository 'deb http://ftp.osuosl.org/pub/mariadb/repo/5.5/ubuntu precise main'
-elif [[ x$1 = xmariadb10 ]]; then
-  add-apt-repository 'deb http://ftp.osuosl.org/pub/mariadb/repo/10.0/ubuntu precise main'
-fi
-
-apt-get update
-apt-get -o Dpkg::Options::=--force-confdef -o Dpkg::Options::=--force-confold -y install mariadb-server libmariadbd-dev
```

Comment:
libmariadb-dev and libmysql-dev would be pretty close. Linking against both is probably a reasonable test.

libmariadbd-dev is the development client libraries not the embedded db.